src/app/projects/states/portfolio/directives/portfolio-welcome-step/portfolio-welcome-step.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'portfolio-welcome-step',
+  templateUrl: 'portfolio-welcome-step.component.html',
+  styleUrls: ['portfolio-welcome-step.component.scss']
+})
+export class PortfolioWelcomeStepComponent {
+  @Input() advanceActiveTab: any;
+  @Input() externalName: any;
```

Comment:
External name is a typed component - so do not use any here.

Reply:
It's a constant value. can we directly use that value?

Reply:
Have a look at the `AboutDoubtfireModal`